CONTRIBUTORS.txt

```diff
@@ -173,3 +173,8 @@ under this name, or we did not manage to find their commits in the history.
 - alain lefroy
 - Mark Gius
 - jarradhope
+- nathannaveen <42319948+nathannaveen@users.noreply.github.com>
+- adam-grant-hendry <59346180+adam-grant-hendry@users.noreply.github.com>
+- Deepyaman Datta <deepyaman.datta@utexas.edu>
+- Batuhan Taskaya <isidentical@gmail.com>
+- Alexander Scheel <alexander.m.scheel@gmail.com>
```

Comment:
There's a bug in contributor-txt, could you move those to the contributors section ? (I did not fix this because I was the only one affected so far, guess now I should fix it 😄 )